bridge-history-api/internal/logic/l2_fetcher.go

```diff
@@ -100,6 +100,11 @@ func NewL2FetcherLogic(cfg *config.FetcherConfig, db *gorm.DB, client *ethclient
 		gatewayList = append(gatewayList, common.HexToAddress(cfg.BatchBridgeGatewayAddr))
 	}
 
+	if common.HexToAddress(cfg.WETHGatewayAddr) != (common.Address{}) {
```

Comment:
then should delete weth gateway in `addressList` and `gatewayList` above.

Reply:
fixed this